CHANGELOG.md

```diff
@@ -3,6 +3,35 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## Windows/Linux/macOS 5.2.0-RC - 2017-12-20
+Updated PECL release packages. Here is the list of updates:
+
+### Added
+- Added support for Ubuntu 17 (requires [MSODBC 17 preview](https://github.com/Microsoft/msphpsql/tree/dev/ODBC%2017%20binaries%20preview))
+- Added support for Debian 9 (requires [MSODBC 17 preview](https://github.com/Microsoft/msphpsql/tree/dev/ODBC%2017%20binaries%20preview))
+
+### Fixed
+- Issue [#555](https://github.com/Microsoft/msphpsql/issues/555) - Hebrew strings truncation (requires [MSODBC 17 preview](https://github.com/Microsoft/msphpsql/tree/dev/ODBC%2017%20binaries%20preview))
+- Issue [#615](https://github.com/Microsoft/msphpsql/issues/615) - Added error handling when fetching varchar(max) as a stream with Always Encrypted
+- Adjusted precisions for numeric/decimal inputs with Always Encrypted
+- Fixed bugs when binding parameters with Always Encrypted
+- Fixed warnings as per Prefast code analysis
+
+### Limitations
+- In Linux and macOS, setlocale() only takes effect if it is invoked before the first connection. The subsequent locale setting will not work
```

Comment:
not sure if you need it here if it's already in aelimitation

Reply:
That's not AE related though ;)

Reply:
oh yea nvm, please go ahead and merge